mainsite/assets/js/jquery-func.js

```diff
@@ -7,12 +7,12 @@
     })
 
     $window.on('load', function () {
-        $body.scrollspy('refresh')
+        $.ready.then(function(){
```

Comment:
why this change?

Reply:
i forgot to take that out... Make a new pr?

Reply:
just undo the change on your local copy of your branch and then do a push. the PR will automatically update when you do that,